### PR TITLE
front: fix stdcm rolling stock image display

### LINKED ISSUE #8289: 1TGVDUPL livery is weird in the tab
### What happened?

Don't know what's going on here:
image

### What did you expect to happen?

The livery

### How can we reproduce it (as minimally and precisely as possible)?

.

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Google Chrome

### OSRD version (top right corner `Account` button > `Informations`)

0f97f32f7509cc9722a1a33bec7462ee22009e82

### PR DESCRIPTION
**Pull request by Akctarus**
closes #8289

### CODE REVIEW

**Review by SharglutDev — COMMENTED** [review]
Except the last comment from @thibautsailly on the issue (padding and gradient), it works great !

File: front/src/styles/scss/applications/stdcm/_home.scss [hunk 1]
```diff
@@ -51,9 +51,28 @@
...
+        .stdcm-consist-title {
+          padding-right: 12px;
+          position: relative;
+        }
+
+        .stdcm-consist-title::after {
+          content: '';
+          background: linear-gradient(270deg, rgba(247, 246, 238, 0) 0%, rgb(235, 235, 234) 40%);
+          position: absolute;
+          top: 0;
```
Comment (theocrsb) [inline front/src/styles/scss/applications/stdcm/_home.scss]: I think `top: 0;` is not usefull

File: front/src/styles/scss/applications/stdcm/_home.scss [hunk 2]
```diff
@@ -51,9 +51,28 @@
...
         width: 322px;
 
+        .stdcm-consist-title {
+          padding-right: 12px;
+          position: relative;
+        }
+
+        .stdcm-consist-title::after {
+          content: '';
+          background: linear-gradient(270deg, rgba(247, 246, 238, 0) 0%, rgb(235, 235, 234) 40%);
```
Comment (theocrsb) [inline front/src/styles/scss/applications/stdcm/_home.scss]: I think it's possible to simplify the first color code

```suggestion
    background: linear-gradient(270deg, transparent 0%, rgb(235, 235, 234) 40%);
```

File: front/src/styles/scss/applications/stdcm/_home.scss (hunk 1 above)
Comment (Akctarus) [reply front/src/styles/scss/applications/stdcm/_home.scss]: fixed

File: front/src/styles/scss/applications/stdcm/_home.scss (hunk 2 above)
Comment (Akctarus) [reply front/src/styles/scss/applications/stdcm/_home.scss]: nice catch, fixed

**Review by theocrsb — APPROVED** [review]
LGTM and tested :)

**Comment by SharglutDev** [comment]
There doesn't seems to be a fading on the left of the image

Capture d’écran 2024-11-26 à 11 35 32

Is it the desired effect @thibautsailly ?

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested

**Pull request merged**